### DISCUSSION TITLE
How to set a table's display value via API

### DISCUSSION THREAD

**Discussion by salim-b** [nocodb/nocodb#7884] [Q&A] [answered]
Is it possible to set a table's [display value](https://docs.nocodb.com/fields/display-value/) via NocoDB's [meta API](https://meta-apis-v2.nocodb.com/)?

Maybe via some subkey in the `meta` key of the [Update Table](https://meta-apis-v2.nocodb.com/#tag/DB-Table/operation/db-table-update) endpoint? If so, how exactly?

**Comment by dstala** [comment, marked as answer]
@salim-b You can use this
https://meta-apis-v2.nocodb.com/#tag/DB-Table-Column/operation/db-table-column-primary-column-set

**Reply by salim-b** [reply]
Thank you @dstala, works like a charm!

I think the [API description](https://github.com/nocodb/nocodb/blob/62414238caac2d11badbb5251fbabf10a6e08133/packages/nocodb-sdk/src/lib/Api.ts#L6130-L6139) could be more clear, though.

Instead of the current

> Create Primary Value
>
> Set a primary value on a given column

something like

> Set column as display value
>
> Mark the given column as the table's [display value](https://docs.nocodb.com/fields/display-value/).

would be more intuitive, I guess.
